prisma-fmt/src/get_config.rs

```diff
@@ -59,7 +59,7 @@ fn get_config_impl(params: GetConfigParams) -> Result<serde_json::Value, GetConf
     if !params.ignore_env_var_errors {
         let overrides: Vec<(_, _)> = params.datasource_overrides.into_iter().collect();
         config
-            .resolve_datasource_urls_from_env(&overrides, |key| params.env.get(key).map(String::from))
+            .resolve_direct_datasource_urls_from_env(&overrides, |key| params.env.get(key).map(String::from))
```

Comment:
I am not sure on this one. I don't know what the url is used for in the CLI. Is it used at all? In which commands? IU think we should find out before changing this.

Reply:
My thinking here is, if we're doing formatting, it's probably done at dev time, so we want the direct URL. I think the only place we don't want `directUrl` to override `url` is for the `QE`, no?

Reply:
ah it's a legacy/bad naming situation: getConfig is not about formatting, it's how the CLI reads datasources in a bunch of commands. I don't think it's a good thing that it does that, but it's been there since the dawn of time.

Reply:
Oh, then I don't really understand what `prisma-fmt` does. Is it ever used in anything other than IE/ME/LSP/formatting?

Reply:
So yeah, we export this from `prisma-fmt`:

```rust
/// Docs: https://prisma.github.io/prisma-engines/doc/prisma_fmt/fn.get_config.html
#[wasm_bindgen]
pub fn get_config(params: String) -> Result<String, JsError> {
    prisma_fmt::get_config(params).map_err(|e| JsError::new(&e))
}
```

but where is it used?

Reply:
It's use in the CLI here https://github.com/prisma/prisma/blob/64a850e1c30edc1daba011581b40afb56d57ae47/packages/internals/src/engine-commands/getConfig.ts#L111

Reply:
`@prisma/prisma-fmt-wasm` really is a "Prisma Engines Wasm module" right now. We wanted to rename it to something "Prisma Schema ..." but never got to it if I remember correctly, right @tomhoule? The first functionality was only `prisma-fmt` related.

Reply:
Further internal discussion about this: https://prisma-company.slack.com/archives/C02FNFLDUS3/p1673003064850679